content/nmdc/src/reference/metag_wrapper_doc.rst

```diff
@@ -0,0 +1,97 @@
+
+.. image:: ../_static/images/reference/workflows/metag_workflow2024.svg
+   :align: center
+   :scale: 25%
+
+Workflow Overview
+-----------------
+The NMDC standardized metagenome workflow leverages parts of JGI's production pipeline for short-read data and consists of: `reads quality control (QC) <https://github.com/microbiomedata/ReadsQC>`_, `metagenome assembly <https://github.com/microbiomedata/metaAssembly>`_, `metagenome annotation <https://github.com/microbiomedata/mg_annotation>`_, `read-based taxonomy <https://github.com/microbiomedata/ReadbasedAnalysis>`_, and binning of population genomes to `generate metagenome-assembled genomes (MAGs) <https://github.com/microbiomedata/metaMAGs>`_ workflows.
+
+The reads QC workflow utilizes rqcfilter2 to trim and filter low quality data from raw metagenome Illumina reads (FASTQ files). The workflow additionally removes artifacts, linkers, adapters, spike-in reads, and reads mapping to several hosts and common contaminants.
+
+The read-based taxonomy classification workflow utilizes GOTTCHA2, Kraken2, and Centrifuge to profile quality-controlled reads to accommodate varied project goals and sequencing approaches that cover a spectrum from high sensitivity to high specificity that is dependent on the algorithms and cut-off levels chosen from different tools. 
+
+The metagenome assembly workflow uses bbcms, metaSPAdes, and BBMap to run error correction, assembly, and assembly validation, respectively. The metagenome annotation workflow takes in assembled metagenomes and generates structural and functional annotations. The MAGs workflow uses metabat2 to generate metagenome bins and applies the MIMAG standards using annotated tRNAs, rRNAs, and marker genes with checkM to estimate completeness and contamination and subsequent taxonomic lineage assignment.
+
+Users can run a single workflow within the metagenome pipeline with the appropriate input files, but the entire metagenome workflow is available to run from start to finish on NMDC EDGE from a single input raw Illumina file.
+
+
+Workflow Availability
+---------------------
+The workflow is available in its individual components on GitHub and as a whole to run on `NMDC-EDGE <https://nmdc-edge.org/home>`_. 
+
+- `ReadsQC <https://github.com/microbiomedata/ReadsQC>`_ 
+- `MetaAssembly <https://github.com/microbiomedata/metaAssembly>`_
+- `MG Annotation <https://github.com/microbiomedata/mg_annotation>`_
+- `MetaMAGs <https://github.com/microbiomedata/metaMAGs>`_ 
+- `Read-based Taxonomy <https://github.com/microbiomedata/ReadbasedAnalysis>`_
+
+
+Requirements for Execution:  
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+(recommendations are in **bold**)
+
+- WDL-capable Workflow Execution Tool (**Cromwell**)
+- Container Runtime that can load Docker images (**Docker v2.1.0.3 or higher**)
+
+Hardware Requirements:
+~~~~~~~~~~~~~~~~~~~~~~
+- Disk space: 258 GB for databases 
+- 86 GB RAM
+
+Workflow Dependencies
+---------------------
+
+Third party software:
```

Comment:
In the sentence following this heading, "third-party" is hyphenated. I recommend hyphenating it here, too. I think hyphenation is grammatically correct in this case (citation goes here... 👀).

```suggestion
Third-party software:
```